libs/Microsoft.MixedReality.WebRTC.Native/include/interop_api.h

```diff
@@ -17,6 +17,24 @@ enum class mrsBool : int32_t { kTrue = -1, kFalse = 0 };
 // Generic utilities
 //
 
+/// Global MixedReality-WebRTC library shutdown options.
+enum class mrsShutdownOptions : uint32_t {
+  kNone = 0,
```

Comment:
nit: Probably needs a comment mentioning possible deadlocks. Also I find a bit surprising that 0 is not the default - should be mentioned too (or the flags be changed so that 0 is the default).

Reply:
Added a `kDefault` to make things explicit, and inverted the meaning to have the flag set activating the dangerous behavior.